README.md

```diff
@@ -1,2 +1,22 @@
-# final-ci-docker
-Docker files for final-ci service.
+# Dockerized final-api application
+
+This solution allows running Dockerized version of [Final-API](https://github.com/AVGTechnologies/final-api).
+
+## What does it mean "dockerised"? What is the motivation?
+Docker is a solution allowing definition of service containers, which can
+be used for comfortable service deployment and cleanup.
+
+This version of final-api targets to bring more comfort to the
+developers.
+
+## How can I use this solution?
+* Clone this repository
```

Comment:
No they will be cloned from master but you still have to provide configuration. On the other hand you can override your sources with mounted volume. Later on we will implement solution, that you can start service from other origins and branches/commits.